lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -203,6 +203,14 @@ export const ClientAuthErrorMessage = {
     keyIdMissing: {
         code: "key_id_missing",
         desc: "A keyId value is missing from the requested bound token's cache record and is required to match the token to it's stored binding key."
+    },
+    noNetworkConnectivity: {
+        code: "no_network_connectivity",
+        desc: "No network connectivity. Check your internet connection."
+    },
+    userCanceledError: {
+        code: "user_canceled",
```

Comment:
@tnorling I noticed this error when we pulled in a new version of MSAL. How does this relate to `BrowserAuthError` `user_cancelled`? The spelling is different and I'm not sure if that's intentional. Is new handling needed for this version of `user_canceled`?